packages/rolldown/build-binding.js

```diff
@@ -0,0 +1,45 @@
+const { spawn } = require('node:child_process')
+const fs = require('fs-extra')
+const glob = require('glob')
+const path = require('node:path')
+
+const cmd = spawn(
+  'npx',
+  [
+    'napi',
+    'build',
+    '-o=./src',
+    '--manifest-path',
+    '../../crates/rolldown_binding/Cargo.toml',
+    '--platform',
+    '-p',
+    'rolldown_binding',
+    '--js',
+    'binding.js',
+    '--dts',
+    'binding.d.ts',
+    '--no-const-enum',
+    '--no-dts-cache',
+  ],
+  {
+    stdio: 'inherit', // Directly inherit stdio (preserves colors)
+    env: { ...process.env, RUSTC_COLOR: 'always' }, // Force color output
+  },
+)
+
+// Inspect exit code
+cmd.on('close', (code) => {
```

Comment:
First I saw this, I was like this solves the problem nicely. But then again, wait a minute, instead of removing files after failing, we could just add the clean up logic in the beginning.

I mean it's nature to clean up the old artifacts before emitting new ones.